### PR TITLE
Update terms to glossary

### PR DESCRIPTION
**Pull request by tab-cmd**
# Overview

Update sequence to inquiry and epoch to series

## Ticket

https://www.pivotaltracker.com/story/show/176097858

## Contributions

Too many to list. Replaced all usage of sequence or seq with inq. Replaced all usage of epoch or epochs with series.

## Test

- Run pytest
- Run all tasks in fake data mode

## Documentaion

- Are documentation updates required? In-line, README, or [documentation](https://github.com/BciPy/bcipy.github.io)? Verify the updates you did here. Updated where appropriate

### CODE REVIEW

File: bcipy/parameters/parameters.json [hunk 1]
```diff
@@ -628,32 +628,32 @@
     "recommended_values": "",
     "type": "str"
   },
-  "min_seq_len": {
+  "min_inq_len": {
     "value": "1",
     "section": "bci_config",
-    "readableName": "Minimum Sequence Length",
-    "helpTip": "The minimum number of sequences to present in spelling tasks",
+    "readableName": "Minimum inquiry Length",
```
Comment (lawhead) [inline bcipy/parameters/parameters.json]: You might want to capitalize 'inquiry' here for consistency.

**Review by lawhead — APPROVED** [review]

**Comment by lawhead** [comment]
Do we have the glossary terms defined anywhere in the code base? If not we may want to include them in one of the READMEs. Also, in our Contribution Guidelines have a bullet point to `Use informative names for functions and classes`. We may want to expand this to mention using the appropriate terminology (as defined in the glossary) when possible.

File: bcipy/parameters/parameters.json (hunk 1 above)
Comment (tab-cmd) [reply bcipy/parameters/parameters.json]: Nice catch - I tried to find them all but I knew a few would slip through!

File: README.md [hunk 2]
```diff
@@ -71,7 +71,7 @@ make dev-install
 
 Start by running `python bcipy/gui/BCInterface.py` in your command prompt or terminal. You may also invoke the experiment directly using command line tools for bci_main.py.
```
Comment (AlisterD) [inline README.md:72]: Would it make sense to create a make command for this as well?

File: bcipy/display/rsvp/demo/demo_copyphrase_rsvp.py [hunk 3]
```diff
@@ -98,29 +98,29 @@
     rsvp.sti.height = sti_height
 
     for idx in range(int(len(ele_sti) / 2)):
-        # Schedule a sequence
-        rsvp.stimuli_sequence = ele_sti[counter]
+        # Schedule a inquiry
+        rsvp.stimuli_inquiry = ele_sti[counter]
```
Comment (AlisterD) [inline bcipy/display/rsvp/demo/demo_copyphrase_rsvp.py:102]: Good change to keep it clean start with definition

**Review by AlisterD — APPROVED** [review]

File: bcipy/helpers/stimuli.py [hunk 4]
```diff
@@ -122,23 +122,23 @@ def best_case_rsvp_seq_gen(alp: list,
                 First element is the target, second element is the fixation
                 Observe that [-1] element represents the trial information
```
Comment (AlisterD) [inline bcipy/helpers/stimuli.py:123]: good explanation here

**Pull request merged**